eclass/meson.eclass

```diff
@@ -57,6 +57,11 @@ BDEPEND=">=dev-util/meson-0.62.2
 # Build directory, location where all generated files should be placed.
 # If this isn't set, it defaults to ${WORKDIR}/${P}-build.
 
+# @ECLASS_VARIABLE: EMESON_VERBOSE
+# @DESCRIPTION:
```

Comment:
If we do this, it should be a user variable, as we never want ebuilds to set it.

Reply:
I didn't make it a user variable because `${CMAKE_VERBOSE}` in `cmake.eclass` also wasn't. (That one probably should be a user variable too though considering `grep` didn't find it in any ebuild.)